ci/summa_install_utils

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
a warning on this (I know we have this still in VIC).

As was pointed out in this https://github.com/travis-ci/travis-ci/issues/6381#issuecomment-236289178:

> Since you are sourcing a bash script that sets errexit ..., the first nonzero exit (from wget) halts the build immediately and you get no further output.

The nice thing about setting `errexit` is that the build stops immediately, the not nice thing is that it is pretty hard to debug in this fashion.